Remotion/Web/Core/UI/Controls/DropDownMenuImplementation/Rendering/DropDownMenuRenderer.cs

```diff
@@ -266,7 +266,7 @@ private void RegisterEventHandlerScripts (DropDownMenuRenderingContext rendering
           : renderingContext.Control.GetSelectionCount);
       string hasDedicatedDropDownMenuElement = renderingContext.Control.Mode == MenuMode.DropDownMenu ? "true" : "false";
       string script = string.Format (
-          "$('#{0}').keydown( function(event){{ DropDownMenu.OnKeyDown(event, $('#{0}'), {1}, {2}); }} );",
+          "$('#{0}').keydown( function(event){{ DropDownMenu.OnKeyDown(event, '#{0}', {1}, {2}); }} );",
```

Comment:
I still smell some jQuery 🐕

Reply:
You can use a regex like `\$|jQuery` to catch most of the remaining jQuery usages that might still linger in C# code

Reply:
How did I miss that 😂